includes/compatibility/class-wcpdf-compatibility-third-party-plugins.php

```diff
@@ -192,6 +195,50 @@ public function add_wpc_product_bundles_classes ( $classes, $document_type, $ord
 		return $classes;
 	}
 
+	/**
+	 * YITH WooCommerce Product Bundles compatibility
+	 *
+	 * @param string $classes CSS classes for item row (tr)
+	 * @param string $document_type PDF Document type
+	 * @param object $order WC_Order order
+	 * @param int $item_id WooCommerce Item ID
+	 */
+	public function add_yith_product_bundles_classes( string $classes, string $document_type, object $order, int $item_id = 0 ): string {
+		if ( ! class_exists( 'YITH_WCPB' ) ) {
+			return $classes;
+		}
+
+		$item_id = ! empty( $item_id ) ? $item_id : $this->get_item_id_from_classes( $classes );
+
+		if ( empty( $item_id ) ) {
+			return $classes;
+		}
+
+		$item = new \WC_Order_Item_Product( $item_id );
+
+		if ( ! is_callable( array( $item, 'get_product' ) ) ) {
+			return $classes;
+		}
+
+		$product = $item->get_product();
+
+		if ( empty( $product ) ) {
+			return $classes;
+		}
+
+		$product_type      = $product->get_type();
+		$is_bundle_product = ( 'yith_bundle' === $product_type );
+		$is_bundle_item    = wc_get_order_item_meta( $item_id, '_bundled_by', true );
+
+		if ( $is_bundle_product ) {
```

Comment:
Maybe you could call the product type here directly?

```php
if ( 'yith_bundle' === $product->get_type() ) {
```

This way you save 2 lines of code because you're not re-using those variables.

Reply:
Thank you for the suggestion.